modules/apps/oauth2-provider/oauth2-provider-test/src/testIntegration/java/com/liferay/oauth2/provider/client/test/RefreshTokenTest.java

```diff
@@ -109,6 +110,36 @@ public void test() throws Exception {
 		}
 	}
 
+	@Test
+	public void testExpiration() throws Exception {
+		JSONObject jsonObject = getToken(
+			"oauthTestApplication", null,
+			getResourceOwnerPasswordBiFunction("test@liferay.com", "test"),
+			this::parseJSONObject);
+
+		Thread.sleep(1500);
```

Comment:
hey @martamedio,
we shoudn't have time dependent tests because these make the CI break from time to time :trollface: 
why does this need this ```sleep```?

Reply:
we should try to wait until the proper conditions are met